frontend/app/common/api.ts

```diff
@@ -44,14 +20,14 @@ const __loginViaEmail = (token: string): Promise<User | null> =>
  * @param address email address
  */
 export const sendEmailVerificationRequest = (username: string, address: string): Promise<void> =>
-  fetcher.get(stringifyUrl('/auth/email/login', { id: siteId, user: username, address }));
+  fetcher.get('/auth/email/login', { id: siteId, user: username, address });
 
 export const logIn = (provider: AuthProvider): Promise<User | null> => {
   if (provider.name === 'anonymous') return __loginAnonymously(provider.username);
   if (provider.name === 'email') return __loginViaEmail(provider.token);
 
   return new Promise<User | null>((resolve, reject) => {
-    const url = stringifyUrl(`${BASE_URL}/auth/${provider.name}/login`, {
+    const url = stringifyUrl(`/auth/${provider.name}/login`, {
```

Comment:
That's the only use of exported `stringifyUrl` I could find, doesn't feel right to expose this internally used function for that single use case. Maybe would be better to create a wrapper `constructLoginURL` and export it? Then it will stay weird, but it will be explicitly weird instead of the current implicit way.